src/uct/ib/base/ib_md.c

```diff
@@ -1192,6 +1192,30 @@ static void uct_ib_md_check_dmabuf(uct_ib_md_t *md)
 #endif
 }
 
+void uct_ib_md_check_odp(uct_ib_md_t *md)
```

Comment:
There is Verbs API to check ODP, https://docs.nvidia.com/networking/display/mlnxofedv492240/optimized+memory+access#src-39263878_OptimizedMemoryAccess-QueryCapabilities.1
why not use it?

Reply:
Am I missing anything or the only API to check it which is present there is `ibv_exp_query_device` which is part of experimental verbs API which was completely removed from UCX some time ago?

Reply:
right, this pointer to exp verbs is irrelevant, pls see https://github.com/linux-rdma/rdma-core/blob/dff9ddeee32e74152230313cb021b1a722cdcc06/libibverbs/verbs.h#L347

Reply:
If I understand the things correctly this structure is already used to check ODP in DEVX related code (see `IBV_DEVICE_HAS_ODP`) but there is only one of many checks that helps us to detect ODP feature and it seems like in DEVX MD case this check is not enough to tell whether ODP available or not. 

So the question is whether checking this field would be enough to be sure that ODP is supported in verbs MD case? And if yes, why it is not enough to check that field only for DEVX MD case?